packages/react-aria-components/stories/Popover.stories.tsx

```diff
@@ -433,3 +433,28 @@ export const PopoverTriggerWidthExample = () => (
     </Popover>
   </DialogTrigger>
 );
+
+const PopoverRightExampleContent = () => {
+  const [s, ss] = useState(false);
+  useEffect(() => {
+    setTimeout(() => {
+      ss(true);
+    }, 1500);
+  });
+  return s ? (
+    <div>
+      Longer content goes here. Hello, World! This is a popover.
+    </div>
+  ) : (
+    <div>Short content goes here.</div>
+  );
+};
+
+export const PopoverRightExample = () => (
+  <MenuTrigger>
+    <Button style={{marginLeft: 'auto'}}>Open popover</Button>
+    <Popover placement="bottom end">
+      <PopoverRightExampleContent />
+    </Popover>
+  </MenuTrigger>
+);
```

Comment:
There are some examples around the code base of how to add a description to the story so we know how to test it in the future. You may need to adjust the code sample I wrote below.
```suggestion
export const PopoverRightExample = {
render: () => (
  <MenuTrigger>
    <Button style={{marginLeft: 'auto'}}>Open popover</Button>
    <Popover placement="bottom end">
      <PopoverRightExampleContent />
    </Popover>
  </MenuTrigger>
),
parameters: {description: 'how to test this story, what does it do?'},
name: 'more descriptive name?'
};
```